data/presets/climbing/route_bottom.json

```diff
@@ -0,0 +1,11 @@
+{
+    "icon": "temaki-climbing",
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "climbing": "route_bottom"
```

Comment:
Lets remove this and first merge #1590 (as you already suggested)

Reply:
I've removed it now.